onwuvic/MyDiary
# #158909833 Create user interface to edit diary

#### What does this PR do?
User interface to edit existing diary

#### Description of Task to be completed?
As a User, I should be able to click on a single entry, so that I can modify it.

#### How should this be manually tested?
Given An entry/diary
When User click on it to modify it
Then User should see a simple descriptive form with the current entry content and button to cancel or save modify content.

#### What are the relevant pivotal tracker stories?
#158909833
